react-big-calendar/index.d.ts

```diff
@@ -48,5 +50,9 @@ declare module 'react-big-calendar' {
         static globalizeLocalizer(globalizeInstance: Object): void;
     }
 
-     export default BigCalendar;
+    /* This enables 'import * as BigCalendar' syntax when compiling to es2015 */
+    namespace BigCalendar {}
+
+    /* react-big-calendar is exported as a commonjs module (it uses babel-preset-jason) */
+    export = BigCalendar;
```

Comment:
@frankwallis please remove namespace, and leave just `export = BigCalendar;`, this way it will be correct with library export and don't need the namespace hack, now you will be able to use default import with --allowSyntheticDefaultImports flag, or require it